africanus/model/spi/examples/README.rst

```diff
@@ -0,0 +1,33 @@
+Simple SPI Fitter
+=================
+
+Fits a simple spectral index model to image cubes. Usage is as follows
+
+$ ./simple_spi_fitter.py --fitsmodel=/path/to/model.fits 
```

Comment:
This is a  nice description of the example. You could introduce some restructured text code formatting here. Either:

```
Fits a simple spectral index model to image cubes. Usage is as follows::

    $ ./simple_spi_fitter.py --fitsmodel=/path/to/model.fits 

```

or 

```
Fits a simple spectral index model to image cubes. Usage is as follows

.. code-block:: bash

    $ ./simple_spi_fitter.py --fitsmodel=/path/to/model.fits 

```
should do it